pandas/formats/format.py

```diff
@@ -2260,6 +2260,35 @@ def _format_strings(self):
         return fmt_values
 
 
+def format_percentiles(percentiles):
+    '''
+    Outputs rounded and formatted percentiles. Rounding preserves uniqeness.
+
+    Parameters
+    ----------
+    percentiles - 1d ndarray of strictly increasing floats from interval [0,1]
```

Comment:
If I sort it here and it's for some reason not in sync with what's inside `describe`, then `describe` will give false output. I'd rather keep the order unchanged. (I'm thinking of this function as something that merely gives a formatted output to another array of floats.)